src/core/solvers/box-ddp.cpp

```diff
@@ -36,6 +41,9 @@ void SolverBoxDDP::allocateData() {
   const std::size_t T = problem_->get_T();
   Quu_inv_.resize(T);
   const std::size_t nu = problem_->get_nu_max();
+#ifdef CROCODDYL_WITH_MULTITHREADING
+#pragma omp parallel for num_threads(NUM_THREADS)
```

Comment:
Does this have a practical benefit? This requires spawning & closing threads for allocating a few matrices (whereas the other parallelisations focused on expensive operations)

Reply:
I agree with Wolf here. It doesn't make sense to use multithreading at memory allocation stage.

Reply:
I removed the multi-threading routine in all cases where we allocate data